tests/ContainerTest.php

```diff
@@ -435,6 +592,54 @@ public function testContainerResolvesRegisteredCallable()
         $this->assertInstanceOf('League\Container\Test\Asset\BazInterface', $foo->bar->baz);
     }
 
+    /**
+     * Assert container resolves a registered named function via call
+     *
+     * @return void
+     */
+    public function testContainerResolvesRegisteredNamedFunction()
+    {
+        $c = new Container;
+
+        $c->invokable('function', '\League\Container\Test\Asset\withArgument')->withArgument('hello');
+
+        $this->assertSame($c->call('function'), 'hello');
+    }
+
+    /**
+     * Asserts that a class method is registered as a callable when aliased as string
+     *
+     * @return void
+     */
+    public function testContainerResolvesRegisteredClassMethodWhenRegisteredAsString()
+    {
+        $c = new Container;
```

Comment:
@philsturgeon not in this particular case as the method called doesn't return the container but a definition. However, there are probably a couple of places I may be able to do so, next iteration will be splitting the tests out in to multiple files so will look through as part of that.